subgraphs/uniswap-v2/src/Factory.ts

```diff
@@ -0,0 +1,61 @@
+// import { log } from '@graphprotocol/graph-ts'
+import { PairCreated } from './../generated/Factory/Factory'
+import { DexAmmProtocol, LiquidityPool, Bundle } from './../generated/schema'
+import { Pair as PairTemplate } from '../generated/templates'
+import { getOrCreateToken, getOrCreateRewardToken } from './common/tokens'
+import {
+  FACTORY_ADDRESS,
+  BIGDECIMAL_ZERO
+} from './common/constants'
+
+export function handleNewPair(event: PairCreated): void {
+
+  let protocol = DexAmmProtocol.load(FACTORY_ADDRESS)
+
+  if (protocol === null) {
+    protocol = new DexAmmProtocol(FACTORY_ADDRESS)
+    protocol.name = "Uniswap v3"
+    protocol.slug = "uniswap-v3"
+    protocol.network = "ETHEREUM"
+    protocol.type = "EXCHANGE"
+
+    // create new bundle
+    let bundle = new Bundle('1')
+    bundle.ethPrice = BIGDECIMAL_ZERO
+    bundle.save()
+  }  
+
+  // create the tokens
+  let token0 = getOrCreateToken(event.params.token0)
+  let token1 = getOrCreateToken(event.params.token1)
+  let rewardToken0 = getOrCreateRewardToken(event.params.token0, "DEPOSIT")
+  let rewardToken1 = getOrCreateRewardToken(event.params.token0, "DEPOSIT")
+
+  let pool = new LiquidityPool(event.params.pair.toHexString())
+
+  pool.protocol = "ETHEREUM"
+  pool.inputTokens =  [token0.id, token1.id]
+  pool.outputToken = token0.symbol + "/" + token1.symbol + "Uniswap v2 LP"
```

Comment:
The `outputToken` ID should be the address of the LP token

Reply:
fixed